library/core/src/commonMain/kotlin/com/attafitamim/krop/core/utils/ZoomLimits.kt

```diff
@@ -0,0 +1,24 @@
+package com.attafitamim.krop.core.utils
+
+import androidx.compose.ui.unit.IntSize
+
+class ZoomLimits(
+    originalImageSize: IntSize,
+    view: IntSize,
+    val minCropSize: Int = 50, // TODO make this a CropperStyle parameter
```

Comment:
I think this would be a good idea to add this parameter to `CropperStyle` so we can customize this behavior